Hw05

amitator opened the pull request:
Fixed everithing from prev PR. 
+Fixed -  QuizImpl.collectQuestions не попал в ПР напишу тут. Вы в данном методе "руками" сзодаете QuestionsDaoImpl, хотя оно @Service и его бы лучше внедрить через @Autowired

amitator committed:
added Locale command.
added bundle.selectBundleFile() to getQuestions.
removed LocaleCommand.

amitator committed:
renamed UserName/UserNameImpl to UserNameRequestService/UserNameRequestServiceImpl

amitator committed:
fixed fileName chose at getQuestions()

amitator committed:
removed reader and added QuestionsDaoImpl

stvort commented on src/main/java/ru/otus/spring/hw01/services/QuizImpl.java:
```diff
@@ -14,28 +13,27 @@
 public class QuizImpl implements Quiz {
 
-    private UserNameImpl userName;
+    private UserNameRequestServiceImpl userName;
```
Тут и далее лучше связывать по интерфейсу

amitator replied:
fixed

stvort commented on src/main/java/ru/otus/spring/hw01/shell/LanguageCommand.java:
```diff
@@ -0,0 +1,38 @@
+public class LanguageCommand {
+    private BundleChoiceDependOnLocale bundle;
+
+    private String language;
```
А зачем она тут?

amitator replied:
fixed

amitator committed:
removed String language

amitator committed:
removed ctx.getBean(QuizImpl.class).run().
changed UserNameRequestServiceImpl to UserNameRequestService and QuestionsDaoImpl to QuestionsDao
